tests/network/jumbo_frame/test_bond.py

```diff
@@ -21,11 +21,13 @@
 BOND_NAME = "jfbond1"
 BRIDGE_NAME = "brbond1"
 
-pytestmark = pytest.mark.usefixtures(
-    "skip_when_no_jumbo_frame_support",
```

Comment:
jumbo frame should be also considered `special_infra`, no?

Reply:
https://github.com/RedHatQE/openshift-virtualization-tests/pull/287#discussion_r1957299310